cmd/s2i/main.go

```diff
@@ -356,28 +358,31 @@ func checkErr(err error) {
 
 func main() {
 	cfg := &api.Config{}
-	stiCmd := &cobra.Command{
+	s2iCmd := &cobra.Command{
 		Use: "s2i",
 		Long: "Source-to-image (S2I) is a tool for building repeatable docker images.\n\n" +
 			"A command line interface that injects and assembles source code into a docker image.\n" +
 			"Complete documentation is available at http://github.com/openshift/source-to-image",
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			if fileLogger, ok := glog.(*utilglog.FileLogger); ok {
+				fileLogger.SetLoggerLevel(*loglevelPtr)
+			}
+		},
```

Comment:
The original goal behind @smarterclayton doing the wrapper was to give us the ability to log without timestamps when s2i is being invoked from within openshift.  Ultimately we'll pick another logging framework that does what we want (ie allows us to better control the prefixing and output target), but i don't think we want to head down the path of implementing our own full logger to get there.